.github/workflows/packages-build.yml

```diff
@@ -0,0 +1,141 @@
+name: Packages build
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        package_type: ["deb", "rpm"]
+        python-version: [3.8]
+        include:
+          - os: windows-2019
+            package_type: "msi"
+            python-version: "3.8"
```

Comment:
Do we need to also specify a specific patch Python version for package reproducibility purposes (e.g. 3.8.x) or it doesn't matter?